x-pack/legacy/plugins/siem/public/components/page/index.tsx

```diff
@@ -34,6 +34,19 @@ export const AppGlobalStyle = createGlobalStyle`
   .euiToolTip {
     z-index: 9950 !important;
   }
+
+  /* 
+    overrides the default styling of euiComboBoxOptionsList because it's implemented
+    as a popover, so it's not selectable as a child of the styled component
+  */
```

Comment:
Thanks for keeping these comments around -- will be beneficial to our future selves! 🙂